src/gui/addnewtorrentdialog.cpp

```diff
@@ -645,8 +645,10 @@ void AddNewTorrentDialog::accept()
     if (ui->comboTTM->currentIndex() != 1) { // 0 is Manual mode and 1 is Automatic mode. Handle all non 1 values as manual mode.
         params.savePath = savePath;
         saveSavePathHistory();
-        if (ui->defaultSavePathCheckBox->isChecked())
+        if (ui->defaultSavePathCheckBox->isChecked()) {
             settings()->storeValue(KEY_DEFAULTSAVEPATH, savePath);
+            BitTorrent::Session::instance()->setDefaultSavePath( savePath );
```

Comment:
Don't use extra space before and after the parameter. It should be `BitTorrent::Session::instance()->setDefaultSavePath(savePath);`